highfidelity/hifi
# WL 21508 - Improve upon Skybox Changer (Updated)

This involved a second PR, due to optimizing the way the Skyboxes are laid out. All of the Skyboxes will now appear at once, with the random button being removed. You are able to scroll up and down the list and choose which Skybox you'd like to have on your domain.

## Test Plan
1. Run Script from URL: [skyboxchanger.js](http://mpassets.highfidelity.com/5fbdbeef-1cf8-4954-811d-3d4acbba4dc9-v1/skyboxchanger.js)
2. Launch the SKYBOX app from the Tablet/Toolbar
3. Confirm that you are able to scroll up and down the list of Skyboxes
4. Confirm that you are able to select each Skybox and momentarily have the Skybox of your rezzable domain change.
5. Confirm that when launching, the same Skyboxes appear and are no longer randomized.
